From 4c919be8e1862d1fc74df48e681b254f5814902d Mon Sep 17 00:00:00 2001
From: Jim Bride <jim.bride@intel.com>
Date: Fri, 3 Aug 2012 15:39:53 -0700
Subject: REVERTME core/pathmap.mk: Add ethernet source

Category: AOSP improvement
Domain: AOSP.Connectivity
Origin: internal
Upstream-Candidate: yes

For: APDEV-459
Depends-Change-Id: Ia089dfd98ce529ad88ba1523927887fdb1987ac5
Depends-Change-Id: Iac46e135b65fa2b5d1f155dfdc99c22dda62eb7b
Change-Id: I0836173813a60c9413e225942d3965a4267b6ed5
Signed-off-by: Markus Grape <markus.grape@intel.com>
---
 core/pathmap.mk |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/core/pathmap.mk b/core/pathmap.mk
index 5dbb7a4..70f6e89 100644
--- a/core/pathmap.mk
+++ b/core/pathmap.mk
@@ -111,6 +111,7 @@ FRAMEWORKS_BASE_SUBDIRS := \
 	    telephony \
 	    wifi \
 	    keystore \
+	    ethernet\
 	 )
 
 #
-- 
1.7.4.1

